CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ IMPROVEMENTS:
  * core: Servers can now service client HTTP endpoints [[GH-3892](https://github.com/hashicorp/nomad/issues/3892)]
  * core: More efficient garbage collection of large batches of jobs [[GH-3982](https://github.com/hashicorp/nomad/issues/3982)]
  * core: Allow upgrading/downgrading TLS via SIGHUP on both servers and clients [[GH-3492](https://github.com/hashicorp/nomad/issues/3492)]
+ * core: Enable periodic health checks for drivers, enabled for Docker to start [[GH-3856](https://github.com/hashicorp/nomad/issues/3856)]
```

Comment:
Perhaps `client` instead of `core`?

Reply:
Enable periodic health checks for drivers, enabled for Docker to start -> Enable periodic health checks for drivers. Initial support only includes the Docker driver.